src/directives/form-errors.directive.spec.ts

```diff
@@ -295,14 +297,17 @@ describe("NgxFormErrorsDirective", () => {
 				formErrorsDirective._controlErrors$.subscribe(mockObserver);
 
 				const errorMessages: object = {
-					minlength: "the value should not contain less than X chars",
-					required: "the field is mandatory"
+					minlength: minlengthErrorMessage,
+					required: requiredErrorMessage
 				};
 
-				mockFormErrorsMessageService.getErrorMessage.and.callFake((errorKey: string, errorGroup: string) => {
-					expect(errorGroup).toBeUndefined();
-					return errorMessages[errorKey] || undefined;
-				});
+				mockFormErrorsMessageService.getErrorMessage.and.callFake(
+					(errorKey: string, _formControlName: string, errorGroup: string) => {
+						expect(errorGroup).toBeUndefined();
+						expect(_formControlName).toBe(formControlName);
+						return errorMessages[errorKey] || undefined;
```

Comment:
Same here... feels like we are cheating and returning only the message we care about :(